### PR TITLE
publish do-you-speak-prestashop-march-2023.md

### PR DESCRIPTION
**Pull request by l-delin**

### CODE REVIEW

**Comment by l-delin** [comment]
Hello, @kpodemski and @coffrep, here's the next Crowdin Monthly :-) Can it be published tomorrow so it's not too late this month?

Thanks 🙌

File: content/news/translation/2023/do-you-speak-prestashop-march-2023.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+aliases: ["/news/do-you-speak-prestashop-march-2023"]
+title: "Do You Speak PrestaShop? – March 2023 Edition"
+subtitle: "Where you'll learn that contributors interviews are back"
```
Comment (coffrep) [inline content/news/translation/2023/do-you-speak-prestashop-march-2023.md]:
```suggestion
subtitle: "Where you'll learn that contributor interviews are back"
```

File: content/news/translation/2023/do-you-speak-prestashop-march-2023.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+Congratulations and a huge thank you to all the contributors! 🎉
+ 
+Of course, this is highlighting the languages that made some progress with new translations; but it does not mean that the languages that are not mentioned here are not active.
+Indeed, some editing and rewriting may be going on, but the percentage of translation would not be modified (since the work happens on strings that are already translated). So let's not forget about the work of other contributors! Thanks to you too!
+
+## Let’s keep in touch!
+
+If you have not joined us on Crowdin yet, [it is never too late](https://crowdin.com/project/prestashop-official)! :wink:
+
+You can also join the #translation channel of the PrestaShop Open Source Slack. On this channel, I share information about the translation project. You will also be informed about the publication of the Crowdin monthly report. Of course, you will also be able to ask questions, suggest ideas, or get in touch with other translators.
```
Comment (coffrep) [inline content/news/translation/2023/do-you-speak-prestashop-march-2023.md]:
```suggestion
You can also join the #translation channel of the PrestaShop Open Source Project's Slack. On this channel, I share information about the translation project. You will also be informed about the publication of the Crowdin monthly report. Of course, you will also be able to ask questions, suggest ideas, or get in touch with other translators.
```

**Review by coffrep — DISMISSED** [review]
Hi @l-delin , just added a few suggestions, let me know of your thoughts

**Comment by l-delin** [comment]
Looks good to me, thanks @coffrep!

**Comment by coffrep** [comment]
Glad I could help. I can't publish so I'll let @kpodemski give his final approval - thanks Krystian!

File: content/news/translation/2023/do-you-speak-prestashop-march-2023.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+
+## Project news
+
+### 🎤 Contributor interviews are back!
+
+You’ve read that right – contributor interviews are back! 
+The first interview sheds light on PrestaShop ambassador, merchant and contributor [AmirRzd](https://crowdin.com/profile/amirrzd), who’s been contributing to the translation project for nearly 8 years. 
+
+If you wish to learn more, check out the article [here](https://build.prestashop-project.org/news/2023/contributor-interview-amirhossein/).
```
Comment (kpodemski) [inline content/news/translation/2023/do-you-speak-prestashop-march-2023.md]:
```suggestion
If you wish to learn more, check out the article [here]({{< relref "/news/posts/2023/contributor-interview-amirhossein" >}}).
```

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
thanks @l-delin :)

**Pull request merged**